schema.md

```diff
@@ -158,11 +158,64 @@ same database, the duplicate entry could be written using only the `id`,
 The `related` field gives a list of IDs of closely related vulnerabilities, such
 as the same problem in alternate ecosystems. 
 
-### package field
+### summary, details fields
+
+The `summary` field gives a one-line, English textual summary of the
+vulnerability. It is recommended that this field be kept short, on the order of
+no more than 120 characters.
+
+The `details` field gives additional English textual details about the
+vulnerability. The field is plain text. Newline characters must be considered
+line breaks when displaying the text, but the display need not use a fixed-width
+font, and the text should not assume one.
+
+The `summary` field is plain text.
+
+The `details` field is CommonMark markdown (a subset of GitHub-Flavored
+Markdown). Display code may at its discretion sanitize the input further, such
+as stripping raw HTML and links that do not start with http:// or https://.
+Databases are encouraged not to include those in the first place. (The goal is
+to balance flexibility of presentation with not exposing vulnerability database
+display sites to unnecessary vulnerabilities.)
+
+### affected fields
+
+The `affected` field is a JSON array containing objects that describes the
+affected packages versions, meaning those that contain the vulnerability.
+
+Within each object in the `affected` array, the `package` field identifies the
+package containing the vulnerability. In most cases, there should be exactly one
+entry in the `affected` array per affected `package` to describe all affected
+versions. In rare cases, for example if the `ecosystem_specific` encodes
+platform information that doesn't apply equally to all listed versions and
+ranges, a separate entry with the same `package` in the `affected` array may be
+used.
+
+The `versions` field can enumerate a specific set of affected versions, and the
+`ranges` field can list ranges of affected versions, under a given defined
+ordering. A version is considered affected if it lies within any one of the
+ranges or is listed in the versions list.
+
+The `versions` list should - with one exception - always be present, to allow
```

Comment:
Still, this discussion would get a lot simpler. No exceptions.

Reply:
ack.